README.md

```diff
@@ -12,46 +12,32 @@ We deploy our services into one of two namespaces:
 - `production`: Services for our production servers
 
 ### Template system
-All configurations for our projects can be found under the sites folder. Using the konf.py script, we generate the following Kubernetes objects:
+Using the konf.py script, we generate the following Kubernetes objects:
 - service
 - deployment
 - ingress
 
 E.g.:
 ``` bash
-./konf.py staging sites/canonical.com.yaml
+./konf.py staging ../microcloud.is/konf/site.yaml
 ```
 
 ## Deploying
 
-### Using create-project
-
-When creating a new project, you can either copy an existing projects values file and modify the file as you want. Here is a document explaining all the possibles values:
-
-- [Values file manual](sites/README.md)
-
-Or use `./create-project`. To auto-generate the files for a new project. Run `./create-project project.name` and the tool will create the config files for you.
-
-For example:
-`./create-project new-site.com`
-
-Will create:
- - /sites/new-site.com.yaml
-
 ### To deploy a new service
 
 E.g. to deploy the snapcraft.io service to staging from scratch:
 
 ``` bash
 # E.g. To deploy the snapcraft.io services to staging
-./konf.py staging sites/snapcraft.io.yaml | kubectl apply --filename -
+./konf.py staging ../microcloud.is/konf/site.yaml | kubectl apply --filename -
```

Comment:
Normally this should be run from within a project, not referring to a project outside. So `konf/site.yaml` would be appropriate.